cmd/skupper/skupper.go

```diff
@@ -171,11 +189,14 @@ func main() {
 		Args:  cobra.NoArgs,
 		Run: func(cmd *cobra.Command, args []string) {
 			cli, _ := client.NewClient(namespace, kubeContext, kubeconfig)
-			err := cli.VanRouterRemove(context.Background())
-			if err == nil {
-				fmt.Println("Skupper is now removed from '" + cli.Namespace + "'.")
-			} else {
+			err := cli.VanSiteConfigRemove(context.Background())
+			if err != nil {
```

Comment:
Is the call to VanRouterRemove necessary if the site config is removed?

Reply:
No, it should not be, as the router deployment should be owned by the site config.